Formula/boost.rb

```diff
@@ -1,22 +1,11 @@
 class Boost < Formula
   desc "Collection of portable C++ source libraries"
   homepage "https://www.boost.org/"
-  revision 2
   head "https://github.com/boostorg/boost.git"
 
   stable do
-    url "https://dl.bintray.com/boostorg/release/1.72.0/source/boost_1_72_0.tar.bz2"
-    mirror "https://dl.bintray.com/homebrew/mirror/boost_1_72_0.tar.bz2"
```

Comment:
Please keep the mirror. I'd also move the url and sha256 between `homepage` and `head` now that the patch is removed.

Reply:
agree, I think the mirror got removed during the `brew bump-formula-pr` command process.